xrpl/asyncio/clients/client.py

```diff
@@ -4,9 +4,15 @@
 from abc import ABC, abstractmethod
 from typing import Optional
 
+from typing_extensions import Final
+
 from xrpl.models.requests.request import Request
 from xrpl.models.response import Response
 
+# The default request timeout duration. If you need more time, please pass the required
```

Comment:
I would reword this to not be addressing someone.

`The default request timeout duration. Set in Client._request_impl to allow more time for longer running commands.`